server/lib/routes/get-fxa-client-configuration.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+module.exports = function (config) {
+  var route = {};
+  route.method = 'get';
+  route.path = '/.well-known/fxa-client-configuration';
+
+  var fxaClientConfig = {
+    /*eslint-disable camelcase */
+    auth_server_base_url: config.get('fxaccount_url'),
+    oauth_server_base_url: config.get('oauth_url'),
+    profile_server_base_url: config.get('profile_url'),
+    sync_tokenserver_base_url: config.get('sync_tokenserver_url'),
+  };
+
+  var cacheControlHeader = 'public, max-age=';
+  cacheControlHeader += config.get('fxa_client_configuration.max_age');
```

Comment:
This has (I think) a default value of "1 day" in the config; do you mean that we should also set a default value here in the code in case the config is busted?